bridge/qb/client/functions.lua

```diff
@@ -86,56 +121,111 @@ end
 
 -- Getters
 
----@deprecated use GetVehicles from imports/utils.lua
-functions.GetVehicles = GetVehicles
+---@deprecated use the GetGamePool('CVehicle') native directly
+functions.GetVehicles = function()
+    GetGamePool('CVehicle')
+end
 
----@deprecated use GetObjects from imports/utils.lua
-functions.GetObjects = GetObjects
+---@deprecated use the GetGamePool('CObject') native directly
+functions.GetObjects = function()
+    GetGamePool('CObject')
+end
 
----@deprecated use GetPlayersInScope from imports/utils.lua
+---@deprecated use the GetPlayersInScope native directly
```

Comment:
There is no GetPlayersInScope native

Reply:
Whats the plan for this one @D4isDAVID

Reply:
I think I meant to write `GetActivePlayers` which this function uses internally.